### PR TITLE
Move bamtools to IUC, fix collection handling.

### PR DESCRIPTION
**Pull request by davebx**
Previously bamtools would reduce over a collection. This was corrected for -filter in https://github.com/galaxyproject/tools-devteam/pull/467 but the other two tools were also behaving incorrectly. Additionally, it seems to be the general consensus that tool development should be kept to tools-iuc.

### CODE REVIEW

**Comment by jmchilton** [comment]
Has https://github.com/galaxyproject/tools-devteam/pull/468 this been integrated?

**Comment by davebx** [comment]
@jmchilton no, but it should take long to pull it in. Would be a shame to lose @jj-umn's contribution.

**Comment by davebx** [comment]
Thanks for the work, @jj-umn

File: tool_collections/bamtools/bamtools_split/bamtools_split_mapped.xml [hunk 1]
```diff
@@ -0,0 +1,114 @@
+<<<<<<< HEAD:tool_collections/bamtools/bamtools_split/bamtools-split-mapped.xml
```
Comment (bgruening) [inline tool_collections/bamtools/bamtools_split/bamtools_split_mapped.xml]: alert :)

File: tool_collections/bamtools/bamtools_split/bamtools_split_tag.xml [hunk 2]
```diff
@@ -0,0 +1,75 @@
+<<<<<<< HEAD:tool_collections/bamtools/bamtools_split/bamtools-split-tag.xml
```
Comment (bgruening) [inline tool_collections/bamtools/bamtools_split/bamtools_split_tag.xml]: many more ...

File: tool_collections/bamtools/bamtools_split/bamtools_split_mapped.xml (hunk 1 above)
Comment (davebx) [inline tool_collections/bamtools/bamtools_split/bamtools_split_mapped.xml]: How rude, git told me all was well.

**Comment by bgruening** [comment]
@jj-umn can you please look at this?

**Comment by bgruening** [comment]
Thanks @davebx!

**Comment by jj-umn** [comment]
@davebx @bgruening   Looks good to me.   I downloaded the branch and tested it on some own my own data.   I suggest we merge.

**Comment by jj-umn** [comment]
@bgruening   Should this be a squash and merge?

File: tool_collections/bamtools/bamtools/.shed.yml [hunk 3]
```diff
@@ -0,0 +1,12 @@
...
+- Sequence Analysis
+- SAM
+description: A collection of tools for manipulation of BAM files
+long_description: |
+  BAMTools is a collection of utilities for splitting BAM files.
+  It is based on BAMtools suite of tools by Derek Barnett.
+name: bamtools
+owner: devteam
+homepage_url: https://github.com/pezmaster31/bamtools
+remote_repository_url: https://github.com/galaxyproject/tools-devteam/tree/master/tool_collections/bamtools/bamtools
```
Comment (bgruening) [inline tool_collections/bamtools/bamtools/.shed.yml]: This should be tools-iuc ...

File: tool_collections/bamtools/bamtools_filter/.shed.yml [hunk 4]
```diff
@@ -0,0 +1,12 @@
...
+- Sequence Analysis
+- SAM
+description: A tool for filtering on BAM files
+long_description: |
+  Utility for filtering BAM files. It is based on BAMtools suite
+  of tools by Derek Barnett.
+homepage_url: https://github.com/pezmaster31/bamtools
+name: bamtools_filter
+owner: devteam
+remote_repository_url: https://github.com/galaxyproject/tools-devteam/tree/master/tool_collections/bamtools/bamtools_filter
```
Comment (bgruening) [inline tool_collections/bamtools/bamtools_filter/.shed.yml]: tools-iuc

File: tool_collections/bamtools/bamtools_split/.shed.yml [hunk 5]
```diff
@@ -0,0 +1,20 @@
...
+- Sequence Analysis
+- SAM
+description: Tool for splitting BAM files
+long_description: |
+  Utility for splitting BAM files. It is based on BAMtools suite
+  of tools by Derek Barnett.
+homepage_url: https://github.com/pezmaster31/bamtools
+name: bamtools_split
+owner: devteam
+remote_repository_url: https://github.com/galaxyproject/tools-devteam/tree/master/tool_collections/bamtools/bamtools_split
```
Comment (bgruening) [inline tool_collections/bamtools/bamtools_split/.shed.yml]: tools-iuc

File: tool_collections/bamtools/bamtools_split/.shed.yml [hunk 6]
```diff
@@ -8,7 +8,7 @@ long_description: |
 homepage_url: https://github.com/pezmaster31/bamtools
 name: bamtools_split
 owner: devteam
```
Comment (jj-umn) [inline tool_collections/bamtools/bamtools_split/.shed.yml]: iuc

**Comment by bgruening** [comment]
@jj-umn normal merge is ok. I think it should stay as `devteam`, we do not want to have complete new repositories in the TS, do we?

**Comment by jj-umn** [comment]
@bgruening   No.   I erroneously thought that we had to have iuc ownership, but maintaining tool continuity is better.

**Pull request merged**